rpm/get_version_string.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Bash lib to know the RPM version and revision from a GitHub repository
```

Comment:
Licence header missing.

Reply:
All of the iotagents has the same missed header.

Reply:
Added license header in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/c4b64018e59a70afef77312d670c787539d6d009